File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
// 250724

class Solution {
    public int[] solution(int[] array) {
        int[] answer = {};
        
        answer = new int[2];
        int max = Integer.MIN_VALUE, maxIndex=0;
        
        for(int i=0; i<array.length; i++) {
            if (array[i] > max) {
                max = array[i];
                maxIndex = i;
            }
        }
        
        answer[0] = max;
        answer[1] = maxIndex;
        
        return answer;
    }
}

File: 프로그래머스/0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.py
# 260204

def solution(array):
    answer = []
    max = 0
    max_i = 0
    
    for i in range(len(array)):
        if array[i] > max:
            max = array[i]
            max_i = i
    
    answer.append(max)
    answer.append(max_i)
    
    return answer